static/css/styles.css

```diff
@@ -287,6 +287,14 @@ input {
 	border-bottom-style: solid;
 }
 
+.help-tooltip {
```

Comment:
Hmm, the tooltip is not coming up for me when I hover. Also, what happens for mobile clients? 

Instead of a tooltip, perhaps we should create a FAQ entry that explains what lenses do, and how the url classification is done (via Wappalyzer)?

Reply:
+1 to FAQ, I'll have the icon link to that instead of a tooltip

Reply:
Added PTAL

Reply:
👍